### DISCUSSION TITLE
I have been trying to use cava but cant get it to work. Installed all the prerequisites which were stated on their github and downloaded it from aur but it always shows this error when running the 'cava' command.

### DISCUSSION THREAD

**Discussion by rishav12s** [prasanthrangan/hyprdots#809] [Q&A] [answered]
![240130_07h32m01s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/157917453/3afd07e8-6452-478b-a9ed-08b95e088717)

**Comment by kRHYME7** [comment]
try to set the audio source.

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/8bc00f07-829b-4174-93f9-3c12c76cb155)

**Reply by rishav12s** [reply]
what will be the source for pulse ?

**Reply by kRHYME7** [reply, marked as answer]
might be 'auto' I just uncomment that then it works. 

Not sure what is the fixed for that error but try to uncomment  those pairs.

**Reply by rishav12s** [reply]
Thanks man it works just need to remove the ";" btw one question is cava already synced with wallbash ? like it gets the colour based on wallbash ?

**Reply by kRHYME7** [reply]
Not yet on the repo but I think I managed to make one.  It's easy

**Reply by rishav12s** [reply]
nice could you tell me how to do it ?

**Reply by kRHYME7** [reply]
Sure! How about posting a Wallbash Thread in the discussion? Hoping someone might share their configs too! I'll post it here https://github.com/prasanthrangan/hyprdots/discussions/811 .

**Reply by kRHYME7** [reply]
lmao I did notice my cava .dcol is just the whole config file. I can't post it there yet but here is the file . On the thread I added some instructions how this work

the dcol:
https://raw.githubusercontent.com/kRHYME7/Hyprdesk/Hyprdesk/Configs/.config/hypr/wallbash/cava.dcol
the script: 
https://github.com/kRHYME7/Hyprdesk/blob/Hyprdesk/Configs/.config/hypr/scripts/wallbashcava.sh
What you should focus on is this part 
```
[color]

# Colors can be one of seven predefined: black, blue, cyan, green, magenta, red, white, yellow.
# Or defined by hex code '#xxxxxx' (hex code must be within ''). User defined colors requires
# a terminal that can change color definitions such as Gnome-terminal or rxvt.
# default is to keep current terminal color
; background = default
; foreground = default

# SDL and sdl_glsl only support hex code colors, these are the default:
; background = '#111111'
; foreground = '#33ffff'


# Gradient mode, only hex defined colors are supported,
# background must also be defined in hex or remain commented out. 1 = on, 0 = off.
# You can define as many as 8 different colors. They range from bottom to top of screen
 gradient = 1
 gradient_count = 8
 gradient_color_8 = '#<wallbash_pry1>'
 gradient_color_7 = '#<wallbash_1xa1>'
 gradient_color_6 = '#<wallbash_1xa2>'
 gradient_color_5 = '#<wallbash_1xa3>'
 gradient_color_4 = '#<wallbash_pry2>'
 gradient_color_3 = '#<wallbash_2xa1>'
 gradient_color_2 = '#<wallbash_2xa2>'
 gradient_color_1 = '#<wallbash_2xa3>'
```

Preview (Not really good as Im just testing it) : Also needs a script to reload the config. 
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/f041146b-d6de-429c-8d13-20ddde0f9794)

**Reply by rishav12s** [reply]
still great work its only missing the script for now i hope you will finish it soon

**Reply by kRHYME7** [reply]
@rishav12s  I added a Merge Request recently https://github.com/prasanthrangan/hyprdots/pull/873 Check it out!

**Reply by rishav12s** [reply]
thanks man great work as usual btw (i am going off topic here) could you share the "cat_30.png" wallpaper in the screenshot of the other pr that preview of the wallpaper looks dope ngl

**Reply by kRHYME7** [reply]
Will it broke If I share it directly here?  I forgot where did I snatch this

![cat_30](https://github.com/prasanthrangan/hyprdots/assets/53417443/f443d23c-13cb-43bb-bca2-d1541b2e21d7)
